src/components/ComponentPages/HistoryContainer/Collapse/index.tsx

```diff
@@ -228,7 +228,7 @@ function HistoryCollapse({
                       Object
                     </Button>
                     <Modal
-                      title="Why can't i object?"
+                      title="Why can't I object?"
```

Comment:
I would suggest, move it to common message file

Reply:
Done